src/construction.cpp

```diff
@@ -1500,3 +1500,24 @@ std::vector<std::string> construction::get_folded_time_string( int width ) const
     std::vector<std::string> folded_time = foldstring( time_text, width );
     return folded_time;
 }
+
+void finalize_constructions()
+{
+    std::set<itype_id> frames;
+    for( auto &vp : vpart_info::get_all() ) {
+        if( !vp->has_flag( "INITIAL_PART" ) ) {
+            continue;
+        }
+        frames.insert( vp->item );
```

Comment:
Several parts have the same base item and each item should appear at most once in the requirement vector.

But `done_vehicle` can not handle that case anyway (it will chose the first vehicle part that has the flag and has the consumed item as base). I removed the set and put things directly in the vector. At worst, the same item shows up several times as alternative, which will probably work fine and also indicate the problem.